src/store/store.ts

```diff
@@ -1205,7 +1205,7 @@ export const useStore = defineStore('store', {
         waiveFees: false
       }]
       const haveFilingChange = (
-        (this.isCorrectionFiling && this.hasCorrectionDataChanged) ||
+        (this.isCorrectionFiling) ||
```

Comment:
Add a comment that we always want to show the Fee Summary if this is a correction filing.